activity.c

```diff
@@ -282,7 +282,8 @@ struct activity paging_act = {
 #endif
 #if defined(SOURCE_SAR) || defined(SOURCE_SADF)
 	.hdr_line	= "pgpgin/s;pgpgout/s;fault/s;majflt/s;"
-		          "pgfree/s;pgscank/s;pgscand/s;pgsteal/s",
+			  "pgfree/s;pgscank/s;pgscand/s;pgsteal/s;"
+			  "promote/s;demote/s",
```

Comment:
Metrics start with `pg` when they represent a number of pages. So to be consistent with other fields names, I would rather use `pgprom/s` and `pgdem/s`.